Sources/TokamakCore/Views/ViewBuilder.swift

```diff
@@ -25,13 +25,33 @@ public struct EmptyView: View {
 }
 
 // swiftlint:disable:next type_name
-public enum _ConditionalContent<TrueBranch, FalseBranch>: View
-  where TrueBranch: View, FalseBranch: View {
-  case trueBranch(TrueBranch)
-  case falseBranch(FalseBranch)
+public struct _ConditionalContent<TrueContent, FalseContent>: View
+  where TrueContent: View, FalseContent: View {
+  enum Storage {
+    case trueContent(TrueContent)
+    case falseContent(FalseContent)
+  }
 
-  public var body: Never {
-    neverBody("_ConditionalContent")
+  let storage: Storage
+
+  @ViewBuilder
+  public var body: some View {
+    switch storage {
+    case let .trueContent(view):
+      view
+    case let .falseContent(view):
+      view
+    }
+  }
+}
+
+extension Optional: View where Wrapped: View {
+  public var body: AnyView {
```

Comment:
Please add a FIXME that referers to the relevant issue, much better chance we don't forget about it then.